docs/configuring-services.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: docs
+title: "MicroK8s command reference"
+permalink: /docs/configuring-services
+---
+
+# Configuring MicroK8s services
+
+MicroK8s brings up Kubernetes as a number of different services run through
+`sytemd`. The configuration of these services is read from files stored
+in the $SNAP_DATA directory, which normally points to
+`/var/snap/microk8s/current`.
+
+To reconfigure a service you will need to edit the corresponding file and then
+restart the respective daemon. For example, to add `debug` level logging to
+containerd:
+
+```bash
+echo '-l=debug' | sudo tee -a /var/snap/microk8s/current/args/containerd
+sudo systemctl restart snap.microk8s.daemon-containerd.service
+```
+
+The following systemd services will be run by MicroK8s:
+
+
+### snap.microk8s.daemon-apiserver
+
+The Kubernetes API server validates and configures data for the API objects
+which include pods, services, replication controllers, and others. The API
+Server services REST operations and provides the frontend to the cluster’s
+shared state through which all other components interact.
+
+The service configuration is described in full in the upstream  
+[kube-apiserver documentation][kube-apiserver].
+
```

Comment:
I would add here where the configuration file is as we do for the rest of the services below.

Reply:
done